postcss.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  plugins: {
+    'postcss-import': {
+      root: __dirname,
+    },
+    'postcss-mixins': {},
+    'postcss-each': {},
+    'postcss-cssnext': {}
+  },

```

Comment:
Actualy I tried a lot of options! I tried to specify `root`, `path` (even added `nod_modules/react-toolbox/lib` here, not just `nod_modules`), `addModulesDirectories`, than I added 3 or 4 extra deps to allow better resolving, but it all did not work.